botdemo/app/build.gradle

```diff
@@ -31,18 +29,26 @@ android {
 
     defaultConfig {
         applicationId "nanorep.com.botdemo"
-        minSdkVersion 19
+        minSdkVersion 16
         targetSdkVersion 28
         versionCode 1
-        versionName "1.3.0"
+        versionName "3.6.1.rc1"
         testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+        multiDexEnabled true
+
     }
 
+    testBuildType "sdktesting"
+
     buildTypes {
         release {
             minifyEnabled false
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
+        sdktesting {
```

Comment:
why do u need this - thats for the unit testing

Reply:
removed